ci/docker-pull-or-build-and-push.sh

```diff
@@ -31,7 +38,7 @@ retry 3 docker-compose --ansi never pull $image_name
 # (since v1.25.1, `docker-compose pull` is NOT failing when it can't fetch the image).
 image_tag=$(docker-compose --ansi never config | yq eval ".services.$image_name.image" -)
 docker image inspect "$image_tag" &>/dev/null || {
-  docker-compose --ansi never build --build-arg user_id="$(id -u)" --build-arg group_id="$(id -g)" $image_name
+  docker-compose --ansi never build --build-arg user_id=$USER_ID --build-arg group_id=$GROUP_ID $image_name
```

Comment:
@PawelLipski It might be even better now, i tried to implement solution #3 from https://faun.pub/set-current-host-user-for-docker-container-4e521cef9ffc, setting user with docker-compose

Reply:
Wow I've never thought of mounting `/etc/` files from host into the container... definitely quite convenient, although a bit dubious from security PoV... but given that it's only gonna be used for CI and local development runs, that's probably acceptable!

Reply:
(i.e. this image is not going to be used in a production mode in a cluster etc.)

Reply:
Anyway, pls open yet another PR for that solution so that we have all the options PoCed separately